lib/ui/painting/image_encoding.h

```diff
@@ -5,16 +5,29 @@
 #ifndef FLUTTER_LIB_UI_PAINTING_IMAGE_ENCODING_H_
 #define FLUTTER_LIB_UI_PAINTING_IMAGE_ENCODING_H_
 
+#include "third_party/skia/include/core/SkImage.h"
 #include "third_party/tonic/dart_library_natives.h"
 
 namespace flutter {
 
 class CanvasImage;
 
+// This must be kept in sync with the enum in painting.dart
+enum ImageByteFormat {
+  kRawRGBA,
+  kRawStraightRGBA,
```

Comment:
I know these were just moved but these enum names are incomprehensible 😕

Reply:
Yea, I agree.  They are mirroring the dart api that grew organically.  I'll write up an issue for adding `kStrictRawStraightRGBA` to clear everything up /s.

Reply:
If we really put in the effort and apply ourselves, we'll have enum names as incomprehensible as `VK_FORMAT_G10X6_B10X6_R10X6_3PLANE_422_UNORM_3PACK16`!